zipline/data/bundles/core.py

```diff
@@ -472,7 +472,7 @@ def most_recent_data(bundle_name, timestamp, environ=None):
         environ : dict, optional
             An environment dict to forward to zipline_root.
         """
-        if bundle_name not in bundles:
+        if bundle_name not in bundles and bundle_name != 'test':
```

Comment:
This change is because there is no longer a 'test' bundle, so I used the 'quantopian-quandl' bundle in order to run the script that updates the `test_examples` data (`python tests/resources/rebuild_example_data`). This changed because of zeroing out the treasury data.